TechAndCheck/hypatia
# move from firefox to chrome for screenshotting

This moves the screenshotting of Twitter from Firefox to Chrome. The reasoning is two fold:
1. We're already using Chrome, so might as well
2. There were some concerns that Firefox isn't working too well

To Test:
1. `bundle install`
2. `rails t test/media_sources/twitter_media_source.rb`
3. Check your S3 bucket to make sure the uploads work

Everything should just work.

Note: I'm going to quash a bunch of these commits before merging, but just look at the code.
